### ISSUE TITLE
Unmanaged channel with same name as group displayed as though managed by group

### ISSUE THREAD

**Issue by mike-cr** [urbit/urbit#4294]
**Describe the bug**
If you have an unmanaged chat/notebook/collection named `test`, and then create a group named `test`, the unmanaged channel is displayed as though it were managed by that group. If you actually click on the channel in the group, it takes you to the DMs page and displays it there, so it doesn't seem like it's *actually* managed - just displays as such
 
**To Reproduce**
1. create unmanaged `test` chat
2. create `test` group
3. observe `test` chat gone from DMs and displays under `test` group
4. click on `test` chat under `test` group and observe it takes you to the DMs area

**Expected behavior**
unmanaged channel should not be eaten by group unless you explicitly groupify it

**Desktop (please complete the following information):**
 - OS: arch linux
 - Browser qutebrowser
 - Base hash fienr

**Comment by matildepark** [comment]
I don’t think this is an issue for much longer, we patched the namespacing a while ago. I wonder if we could check this in sidebar though.
